File: app/src/main/java/com/milnest/tasklist/ImgTaskListItem.java
package com.milnest.tasklist;

import android.graphics.Bitmap;

/**
 * Created by t-yar on 17.04.2018.
 */

public class ImgTaskListItem extends TaskListItem {
    //private int mImage;
    private Bitmap mImage;

    public ImgTaskListItem(int id, String name, Bitmap image) {
        super(id, name, TaskListItem.TYPE_ITEM_IMAGE);
        mImage = image;
    }

    public Bitmap getImage() {
        return mImage;
    }

    public void setImage(Bitmap image) {
        mImage = image;
    }
}

File: app/src/main/java/com/milnest/tasklist/ImgTaskListItem.kt
package com.milnest.tasklist

import android.graphics.Bitmap

/**
 * Created by t-yar on 17.04.2018.
 */
/**Класс задачи-картинки
 */
class ImgTaskListItem(id: Int, name: String, //private int mImage;
                      var image: Bitmap?) : TaskListItem(id, name, TaskListItem.TYPE_ITEM_IMAGE)
